pyevtk/hl.py

```diff
@@ -46,7 +51,9 @@ def _addDataToFile(vtkFile, cellData, pointData, fieldData=None):
     if pointData:
         keys = list(pointData.keys())
         # find first scalar and vector data key to set it as attribute
-        scalars = next((key for key in keys if isinstance(pointData[key], np.ndarray)), None)
+        scalars = next(
```

Comment:
This is still just a formatting change, right?

Reply:
It is.